### ISSUE TITLE
[Request] Gravity Rush Native Framebuffer Patch

### ISSUE THREAD

**Issue by Darthsternie** [Electry/VitaGrafix#28]
Hi. It would be really nice if you could add Gravity Rush Native Framebuffer patching to this plugin :)
Not sure how helpful this is but there is a Thread at GBAtemp for manually patching the eboot :)
https://gbatemp.net/threads/gravity-rush-native-framebuffer-size.475088/
Thanks a lot in advance

**Comment by Darthsternie** [comment]
If you require the Eboot.bin.elf or want me to test something just hit me up ^^

**Comment by Electry** [comment]
Hey, I already looked at the gbatemp thread, and that patch only changes the UI res. The game is still rendered at 720x408.

I could get it to render at 960x544, but there were some graphical glitches (like shifted double image).

![2018-08-18-155927](https://user-images.githubusercontent.com/12598379/44299997-0ab8a280-a300-11e8-860a-3a6e3448a123.png)

**Comment by Darthsternie** [comment]
Thanks for the fast reply. Sucks that the game behaves like that with a higher resolution :(
It would still be nice if you could include the UI Res patch. It also has some quirks (the pictures in the readable story are shifted) but atleast from my point of view I found the UI rendered in native res to look nicer than the original one :)

**Comment by molecopopo** [comment]
If it could reach at least 60 fps would be nice

**Comment by ExtremeCaliber** [comment]
If it's overclocked I'm sure it would be able to have a stable frame rate

**Comment by nad45** [comment]
@Electry
And at 800x480, do you still have the graphical glitches?

**Comment by ezzinekaemel** [comment]
Many people are interested in this game, but all requests or issues are simply not being addressed.

**Comment by adrianbass360** [comment]
Yeah, at native res it has many graphical glitches. Has anyone else try the game at different higher resolutions? Eboot file would really be appreciated 😁

**Comment by nad45** [comment]
> Hey, I already looked at the gbatemp thread, and that patch only changes the UI res. The game is still rendered at 720x408.
> 
> I could get it to render at 960x544, but there were some graphical glitches (like shifted double image).
> 
> ![2018-08-18-155927](https://user-images.githubusercontent.com/12598379/44299997-0ab8a280-a300-11e8-860a-3a6e3448a123.png)

Hi @Electry 
Have you found a solution for those glitches?

**Comment by Girotin** [comment]
@Electry Look at all those opened issues for this game;
At least tell us if is it possible to fix / you have interest on fixing this, so we can at least stop thinking that someday this patch will be fixed.

All the job u made till this moment was simply amazing, I don't have words to tell how happy you made me with those graphical patches.
But if you don't wanna work with this anymore, it's ok, I imagine that doing this probably take a good amount of effort, but at least give us an last reply before leaving.

Your job it's absolutely amazing.
If you're going to leave this area, please know that you made a lot of people happy with your work.
You'll have an special place in my heart, and everytime I open an patched game on the Vita I'll remember that you fully worked/have some part on it too.

Thank you so much.

**Comment by Electry** [comment]
Posted a patch in https://github.com/Electry/VitaGrafixPatchlist/issues/21 closing this one as duplicate

**Comment by adrianbass360** [comment]
@Electry NEVER TOO LATE, THANKS!! P.S., I know it's a mess but you able to to release an HD patch for Modracers? Or  maybe Hotshot? Maybe asking for too much over here haha
